tests/test_entity_preprocessor.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+
+from datasets import Dataset
+
+from bsmetadata.preprocessing_utils import EntityPreprocessor
+
+
+class TestEntityPreprocessor(unittest.TestCase):
```

Comment:
Thank you very much for this test! It's really good to understand how the entity extraction works!